.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,45 @@
+name: publish_release_docs
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.GH_BOT_ACCESS_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
+
+    # Sync tag docs to versioned folder
+    - name: publish-tag-to-git
+      uses: ./.github/actions/website-sync
+      id: publish_tag
+      with:
+        repository: grafana/website
+        branch: master
+        host: github.com
+        github_pat: '${{ secrets.GH_BOT_ACCESS_TOKEN }}'
+        source_folder: docs
+        target_folder: 'content/docs/agent/${GITHUB_REF##*/}'
```

Comment:
I _think_ this should work? 🤔 We used to use `${GITHUB_REF##*/}` for automating releases, but I never tried it with a action stage.